tsconfig.json

```diff
@@ -2,20 +2,14 @@
   "compilerOptions": {
     "allowJs": true,
     "baseUrl": "./src",
-    "downlevelIteration": true,
-    "esModuleInterop": true,
     "experimentalDecorators": true,
-    "forceConsistentCasingInFileNames": true,
-    "lib": ["ES2020", "ES2021.Intl"],
+    "lib": ["ES2022"],
     "module": "NodeNext",
     "outDir": "dist/",
-    "paths": {
-      "editor": [".sink.d.ts"]
-    },
     "resolveJsonModule": true,
```

Comment:
Are we importing JSON files anywhere not in a unit test?

Reply:
yeah we use our `package.json` file in a bunch of places